proto/api_test.go

```diff
@@ -150,10 +150,10 @@ func TestCombinable(t *testing.T) {
 
 func TestSetGoErrorCopy(t *testing.T) {
 	rh := ResponseHeader{}
-	err := &Error{Message: "test123"}
-	rh.SetGoError(err)
-	err.Message = "321tset"
-	if rh.Error.Message != "test123" {
+	oErr := &Error{Message: "test123"}
+	rh.Error = oErr
+	rh.SetGoError(&testError{})
```

Comment:
`oErr` is set as `rh.Error` before invoking `SetGoError()`. If `SetGoError()` reused `rh.Error`, it would change `oErr`'s message to "test".